operator/scripts/scaling/scalingAction.sh

```diff
@@ -208,6 +225,30 @@ INPUT
   echo "$num_ms"
 }
 
+# Gets the replicas value from the Cluster. Return -1 if no replicas configured in the Cluster.
+# args:
+# $1 Cluster Custom Resource
+get_replicas_from_cluster() {
+  local clusterJson="$1"
+  local replicas
+  if jq_available; then
+    local numReplicasCmd=".spec.replicas"
+    replicas=$(echo "${clusterJson}" | jq "${numReplicasCmd}" 2>> ${log_file_name} )
+  else
+cat > cmds-$$.py << INPUT
```

Comment:
Perhaps I'm missing something, but it looks like this script leaves behind "cmds-$$.py" files after it runs.

Reply:
They will be cleaned up in line 658

```
# Cleanup cmds-$$.py
[ -e cmds-$$.py ] && rm cmds-$$.py

```